.cirrus.yml

```diff
@@ -41,9 +40,14 @@ global_task_template: &GLOBAL_TASK_TEMPLATE
     folder: "/tmp/ccache_dir"
   depends_built_cache:
     folder: "depends/built"
+    fingerprint_script: echo $CIRRUS_TASK_NAME $(git log -1 --oneline ./depends)
```

Comment:
Might use `git rev-list -1 HEAD ./depends` here; this prints only the (full, independent of `core.abbrev`) commit hash.

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/22710#issuecomment-901260528).